app/views/shared/check_answers/_opponent_details.html.erb

```diff
@@ -1,150 +1,132 @@
 <section class="print-no-break govuk-!-padding-top-6">
-  <%= check_answer_change_link(
-        name: :opponents,
-        url: providers_legal_aid_application_has_other_opponent_path(@legal_aid_application),
-        question: t(".opponent-heading"),
-        read_only:,
-      ) %>
-  <%= govuk_summary_list(card: { title: t(".opponent-heading") }, actions: !read_only) do |summary_list| %>
+  <%= govuk_summary_card(title: t(".opponent-heading"), html_attributes: { id: "app-check-your-answers__opponent" }) do |card| %>
+    <% unless read_only %>
+      <% card.with_action do %>
+        <%= govuk_link_to(t("generic.change"),
+                          providers_legal_aid_application_has_other_opponent_path(@legal_aid_application)) %>
+      <% end %>
+    <% end %>
     <% opponents.order(:created_at).each_with_index do |opponent, i| %>
-      <%= summary_list.with_row(html_attributes: { id: "app-check-your-answers__opponent_#{i + 1}" }) do |row| %>
-        <%= row.with_key(text: "#{t('.opponent')} #{i + 1}", classes: "govuk-!-width-one-half") %>
-        <%= row.with_value { sanitize("#{opponent.full_name},<br>#{opponent_type_description(opponent)}") } %>
+      <%= card.with_summary_list(actions: !read_only) do |summary_list| %>
```

Comment:
action can also be `false`

Reply:
👍 I think the (actions: !read_only) can be removed completedly where the only link is in the card heading will do that for all the partials 🙂

Reply:
👍 have added (actuibs: false) to all applicable card.with_summary_list rows.